### ISSUE TITLE
Download Inventory Report and ABR

### ISSUE THREAD

**Issue by nathankota** [uic-utah/uic-inventory#274]
(_**Note:**_ Inventory Review page and report downloads are only available to elevated (admin) users)

### Inventory status == Submitted
- When `Download` is clicked on the Inventory Review page the inventory report is downloaded to the device as a pdf.

### Inventory status == Approved
- When `Download` is clicked on the Inventory Review page for an approved inventory, 
  - the ABR letter and inventory report are downloaded into a single pdf document

**Comment by steveoh** [comment]
This is implemented in dev. Please validate.

**Comment by nathankota** [comment]
@rsparker-utah, after my review this particular feature works as expected. Please also test and confirm, and close if you agree. 

I found a couple changes to be made to the ABR letters, but that will be for a different issue/task (#282).

**Comment by rsparker-utah** [comment]
@nathankota This looks good to me as well and I'm comfortable with this issue being closed (I don't seem to be able to close this issue on my end). 

These are the steps I took to validate this functionality:

1. I used my public user account to create a new site and submitted an inventory (i.e., [site 120, inventory 216](https://uic-inventory.dev.utah.gov/review/site/120/inventory/216)) to test this functionality. 
2. I then navigated to the Inventory Review page of the submitted inventory using my admin account and clicked the "Download" button. The Inventory Report downloaded as expect (i.e., it only contained the inventory report) and looked great. 
3. I then approved this inventory using my admin account and clicked the "Download" button again on the Inventory Review page. This download also functioned as expected (i.e., it contained both the ABR and Inventory report).  

As a side note, the new "Approve" icon with the approver's information that appears at the top of the screen after inventory approval is so cool!
